stable/mlrun-kit/values.yaml

```diff
@@ -67,25 +83,27 @@ mlrun:
     volumes:
       storageOverride:
         persistentVolumeClaim:
-          claimName: mlrun-pvc
+          claimName: mlrun-api-pvc
+    persistence:
+      enabled: true
+      existingClaim:
+      storageClass:
+      accessMode: "ReadWriteOnce"
+      size: "8Gi"
     extraEnv:
-      - name: MLRUN_HTTPDB__DATA_VOLUME
-        value: "/home/jovyan/data"
-      - name: MLRUN_ARTIFACT_PATH
-        value: "/home/jovyan/data"
-      - name: MLRUN_HTTPDB__REAL_PATH
-        value: "/data"
       - name: MLRUN_HTTPDB__PROJECTS__FOLLOWERS
         value: nuclio
-      - name: MLRUN_STORAGE__AUTO_MOUNT_TYPE
-        value: pvc
-      - name: MLRUN_STORAGE__AUTO_MOUNT_PARAMS
-        value: "pvc_name=mlrun-kit-shared-pvc,volume_mount_path=/home/jovyan/data,volume_name=shared-persistency"
```

Comment:
By removing this, we are disabling the auto_mount feature - where the runs themselves have a mount to the Jupyter's data to access files saved there (python files, datasets, etc...). 
For the lightweight tier this is ok. We'll need to discuss if possible to give those pods read access to the Jupyter's pv for the advanced tiers.

Reply:
for advance we can  configure Jupyter's use s3 bekened , it will require to refactor  Jupyter's  dockerfile, i think we need to open for that different task